client/tts.py

```diff
@@ -129,6 +134,63 @@ def play(self, filename):
         pass
 
 
+class CereprocTTS(AbstractTTSEngine):
+
+    SLUG = "cereproc-tts"
+
+    def __init__(self, personnality, accountid, password):
+        super(self.__class__, self).__init__()
+        self.personnality = personnality
+        self.accountid = accountid
+        self.password = password
+
+    @classmethod
+    def is_available(cls):
+        return (super(cls, cls).is_available() and
+                diagnose.check_python_import('suds.client') and
+                diagnose.check_network_connection())
+
+    @classmethod
+    def get_config(cls):
+        # FIXME: Replace this as soon as we have a config module
+        config = {}
+        # HMM dir
+        # Try to get hmm_dir from config
+        profile_path = jasperpath.config('profile.yml')
+        if os.path.exists(profile_path):
+            with open(profile_path, 'r') as f:
+                profile = yaml.safe_load(f)
+
+                if profile['tts_engine'] == 'cereproc-tts' and \
+                        profile['att-tts']['personnality'] and \
```

Comment:
Since all other tts/stt engines follow the `<name-of-engine>-<type>` scheme (like `google-tts`) and we also have `att-stt` (which is AT&T's STT service), I'd rather rename this to `cereproc-tts`.